docs/installation/sources/sources_windows.rst

```diff
@@ -0,0 +1,241 @@
+.. _windows_sources:
+
+Windows installation from sources
+=================================
+
+To install *eProsima Fast DDS* from sources, you first need to meet the required dependencies
+(see requirements_windows_sources_)
+and then choose whether to follow either the colcon_ (see colcon colcon_installation_windows_) or the CMake_
+(see cmake_installation_windows_) installation instructions.
+
+.. _requirements_windows_sources:
+
+Requirements
+------------
+
+*eProsima Fast RTPS* requires the following dependencies when building from sources in a Windows environment.
+
```

Comment:
In general; please, add a reference label for every title you create

Reply:
ok.